editor/components/post-permalink/style.scss

```diff
@@ -34,3 +38,20 @@
 		@include long-content-fade( $size: 20% );
 	}
 }
+
+.editor-post-permalink__edit-form {
+	width: 100%;
```

Comment:
This element is technically the child of a flex container and could be sized with `flex-basis: 100%; flex-shrink: 1;`

Reply:
💪🏻